sycl/doc/extensions/experimental/sycl_ext_matrix/sycl_ext_oneapi_matrix.asciidoc

```diff
@@ -389,22 +392,30 @@ class tf32;
 } // namespace sycl::ext::oneapi::experimental::matrix::precision
 ```
 
-For example:
+[frame="none",options="header",cols="20%,20%,60%"]
+|======================
+| `joint_matrix` element type | Storage type | Descritpion
+|precision::tf32 | float | The TF32 type has a 19 bit format with one
+sign bit, 8 exponent bits (offering the same range as float), and 10
+mantissa bits (offering the same precision as sycl::half).
+|======================
+
+In order to declare a `joint_matrix` with one of these element types,
+use the abstract type like so:
 
 ```c++
 joint_matrix<sub_group, precision::tf32, use::a, tM, tK,
              layout::row_major> tA;
 ```
 
-Whenever the application loads, stores, fills, or accesses the
-elements of a TF32 matrix, the application sees the elements as
-float. There are special overloads of these functions for TF32 for
-this purpose.
+Operations on these matrices use the functions described above, but
+there are different constraints on the template parameters as
+described below.
 
-==== TF32 load
-These overloads of `joint_matrix_load` load float values into a TF32
-matrix. It is unspecified whether the implementation loads all 32 bits
-into the joint matrix or if it only loads the relevant 19 bits.
+==== load
+The template parameter `T2` must either be the storage format type
+that corresponds to the abstract type `T1` or it must be a
+const-qualified version of that storage format type. For example:
```

Comment:
The text in the sections titled "load", "store", "fill", and "copy" all looks good, but the examples could all be improved.  What you have now is not an example.  Rather, it's a synopsis of the function with some parameters changed.  This synopsis doesn't match the real definition, so I think it makes things more confusing.

It would be better to show a short code snippet that uses each function, like you do in the "Element-wise operations" section".  For example, the "load" example might look like this:

```
joint_matrix<sub_group, precision::tf32, use::a, tM, tK, layout::row_major> tA;

float *buf = /*...*/;
auto pBuf = address_space_cast<sycl::access::address_space::global_space,
                               sycl::access::decorated::no>(buf);

tA.load(g, tA, pBuf, Stride);
```

Reply:
@gmlueck , I agree, I just made the improvement, please review again.